docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -247,6 +247,28 @@ modules are configured only to listen for, and not transmit, the ArmingStatus me
 the ArmingStatus message can be found below in :ref:`dronecan_arming_and_bypass`. Information about configuring your flight controller to transmit the ArmingStatus 
 message can be found in our :ref:`tutorial for integrating with flight controller DroneCAN <dronecan_fc_tutorial>`.
 
+uavcan.equipment.indication.LightsCommand (Data Type ID = 1081)
+###############################################################
+Vertiq modules do not publish this broadcast message, but they do listen for it. The content of this message can be used to dynamically control the white and RGB LEDs
+on Vertiq's LED Boards. For more information on the LED Board, see :ref:`manual_led_support`.
+
+As detailed by the `DroneCAN specification <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#lightscommand>`_, this message contains an array of `SingleLightCommands <https://dronecan.github.io/Specification/7._List_of_standard_data_types/#singlelightcommand>`_
+to be handled by the nodes on the bus, each of which contain a light ID to indicate which light that command is for, and and a field to indicate the desired intensity for the white LED, or the desired color for the RGB LED. For Vertiq modules, 
```

Comment:
don't need the comma before and

and a field that indicates either the intensity of the white LED or color of the RGB LED depending on the targeted LED.